### PR TITLE
Extend ini support

### PR DESCRIPTION
**Pull request by SeptimiuVana**

### CODE REVIEW

File: sdk/src/cameras/itof-camera/mode_info.h [hunk 1]
```diff
@@ -76,6 +77,25 @@ class ModeInfo {
     aditof::Status convertCameraMode(const std::string &modes,
                                      uint8_t &convertedMode);
 
+    /**
+     * Get the sensor width and height for the provided mode
+     * param[in] modes - the name of the mode (represented as a string)
+     * param[out] width - sensor width
+     * param[out] height - sensor height
+     * @return aditof::Status
+     */
```
Comment (dNechita) [inline sdk/src/cameras/itof-camera/mode_info.h:86]: also add description for pixelFormatIndex

File: sdk/src/cameras/itof-camera/mode_info.h [hunk 2]
```diff
@@ -76,6 +77,25 @@ class ModeInfo {
...
+                                       uint16_t *height,
+                                       uint8_t *pixelFormatIndex = nullptr);
+
+    /**
+     * Set number of bits in depth/ab/conf and pixelformat
+     * param[in] control - the name of the parameter
+     * param[in] value - the value
+     * @return aditof::Status
+     */
+    aditof::Status setSensorPixelParam(std::string control, std::string value);
```
Comment (dNechita) [inline sdk/src/cameras/itof-camera/mode_info.h:97]: for efficiency, pass strings by const reference

File: sdk/src/cameras/itof-camera/camera_itof.cpp [hunk 3]
```diff
@@ -1664,6 +1667,15 @@ void CameraItof::configureSensorFrameType() {
         LOG(WARNING) << "partialDepthEnable was not found in .ini file";
     }
 
+    it = m_iniKeyValPairs.find("inputFormat");
+    if (it != m_iniKeyValPairs.end()) {
+        value = it->second;
+        m_depthSensor->setControl("inputFormat", value);
+        ModeInfo::getInstance()->setSensorPixelParam("pixelFormat", value);
+    } else {
+        LOG(WARNING) << "bitsInAB was not found in .ini file";
```
Comment (dNechita) [inline sdk/src/cameras/itof-camera/camera_itof.cpp:1676]: replace "bitsInAB" with "inputFormat"

**Review by dNechita — CHANGES_REQUESTED** [review]

File: sdk/src/cameras/itof-camera/camera_itof.cpp (hunk 3 above)
Comment (SeptimiuVana) [reply sdk/src/cameras/itof-camera/camera_itof.cpp]: Fixed.

File: sdk/src/cameras/itof-camera/mode_info.h (hunk 2 above)
Comment (SeptimiuVana) [reply sdk/src/cameras/itof-camera/mode_info.h]: Fixed, thanks!

File: sdk/src/cameras/itof-camera/mode_info.h (hunk 1 above)
Comment (SeptimiuVana) [reply sdk/src/cameras/itof-camera/mode_info.h:87]: Fixed.

**Review by dNechita — APPROVED** [review]

**Pull request merged**